.github/workflows/build.yml

```diff
@@ -211,6 +211,8 @@ jobs:
       OCTOPUS_CLI_API_KEY: ${{ secrets.OCTOPUS_API_KEY }}
       VERSION: ${{ needs.build.outputs.version }}
     steps:
+    - uses: actions/checkout@v3
```

Comment:
This is needed for `gh cli` to pick up the repository automatically so we don't have to pass it via the command line